services-isochrone/src/main/java/com/mapbox/api/isochrone/MapboxIsochrone.java

```diff
@@ -217,6 +239,23 @@ public Builder addContoursMinutes(@NonNull @IntRange(from = 0, to = 60)
       return this;
     }
 
+    /**
+     * An integer list of meter values to use for each isochrone contour.
+     * The distances, in meters, to use for each isochrone contour. You
+     * must be in increasing order. The maximum distance that can be specified
+     * is 100000 meters (100km).
+     *
+     * @param listOfMeterValues an integer list with at least one number
+     *                           for the meters which represent each contour
+     * @return this builder for chaining options together
+     * @since 7.3.0
+     */
+    public Builder addContoursMeters(@NonNull @IntRange(from = 0, to = 100000)
+                                     Integer... listOfMeterValues) {
```

Comment:
```suggestion
    public Builder addContoursMeters(@NonNull Integer... listOfMeterValues) {
```
Let's not add the integer range, this range can be dynamically changed on the backend.